ambry-api/src/main/java/com/github/ambry/account/AccountCollectionSerde.java

```diff
@@ -58,6 +72,14 @@ public static JSONObject accountsToJson(Collection<Account> accounts) {
     }
   }
 
+  public static Collection<Account> accountsFromInputStreamInJson(ObjectMapper objectMapper, InputStream inputStream)
```

Comment:
Could we have a single static ObjectMapper for accounts that is used for all account/container ser/deser?

Reply:
updated